public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -44,22 +43,19 @@ type ContentPanelProps = {
 
 const ContentPanel = (props: ContentPanelProps) => (
   <EuiPanel
-    style={{
-      padding: '20px 0px',
-      ...props.panelStyles,
-    }}
+    style={{ padding: '20px', ...props.panelStyles }}
     className={props.contentPanelClassName}
   >
     <EuiFlexGroup
-      style={{ padding: '0px 20px', ...props.titleContainerStyles }}
+      style={{ padding: '0px', ...props.titleContainerStyles }}
```

Comment:
I think this will make the line between title and body disconnected to the Panel, have you confirmed with UX?
![Screen Shot 2020-04-30 at 9 24 07 AM](https://user-images.githubusercontent.com/59710443/80734896-bd289d00-8ac4-11ea-959b-ac311554333d.png)